### PR TITLE
Responsiveness Cleanups

### PR DESCRIPTION
**Pull request by slimsolz**

### CODE REVIEW

File: src/components/Charts/StackedBarChart/StackedBarChart.module.scss [hunk 1]
```diff
@@ -16,3 +16,9 @@
     margin-bottom: 20px;
   }
 }
+
+@media only screen and (max-width: 425px) {
+  .StackedBarChart {
+    padding: 12px;
+  }
+}
```
Comment (evie-adenoyin) [inline src/components/Charts/StackedBarChart/StackedBarChart.module.scss:24]: No new line at end of file...

Comment (slimsolz) [reply src/components/Charts/StackedBarChart/StackedBarChart.module.scss]: @evie-adenoyin resolved

**Review by evie-adenoyin — COMMENTED** [review]
LGTM

**Review by evie-adenoyin — COMMENTED** [review]
LGTM

**Pull request merged**